src/PrestaShopBundle/Resources/views/Admin/TwigTemplateForm/prestashop_ui_kit.html.twig

```diff
@@ -23,3 +23,80 @@
  * International Registered Trademark & Property of PrestaShop SA
  *#}
 {% use '@PrestaShop/Admin/TwigTemplateForm/bootstrap_4_horizontal_layout.html.twig' %}
+
```

Comment:
The most important part of solving our wrong bootstrap 4 theme issue (that makes Form customization painful) is here.

We have a lot of options to fix it, like creating a new form theme based on the [official Bootstrap 4 template](https://github.com/symfony/symfony/blob/3.4/src/Symfony/Bridge/Twig/Resources/views/Form/bootstrap_4_horizontal_layout.html.twig), and start to integrate the new pages the "right way".

Once we're happy with our form theme, it will be easy to improve the existing ones.

WDYT @sarjon ?

Reply:
> We have a lot of options to fix it

That's why we should not dive straight into _fixing_, but start with discussion. What options do we have? What should we choose and why? We coud start with an issue I think, wdyt?